vcd/datasource_vcd_data_center_group.go

```diff
@@ -0,0 +1,199 @@
+package vcd
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func datasourceVdcGroup() *schema.Resource {
```

Comment:
The filename `datasource_vcd_data_center_group.go` has not been renamed.

Reply:
thank you, changed